blocks/carousel/carousel.css

```diff
@@ -0,0 +1,488 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+.explore-section {
+    margin-top: 15px;
+    box-sizing: border-box;
+    outline: none;
+}
+
+.explore-section .ideas-strike {
+    background: #FAE4D1;
```

Comment:
About the color code, we can add a property in style.css as common use.

Reply:
Yes please add global tokens for colors. Please refer https://www.icicidirect.com/Content/css/market-style.min.css?ref=0603241106_1